code/game/objects/items/twohanded.dm

```diff
@@ -1016,3 +1016,256 @@
 		C.change_view(CONFIG_GET(string/default_view))
 		user.client.pixel_x = 0
 		user.client.pixel_y = 0
+
+/obj/item/twohanded/electrostaff
+	icon = 'icons/obj/estaff.dmi'
+	icon_state = "electrostaff_3"
+	item_state = "electrostaff_3"
+	lefthand_file = 'icons/mob/inhands/weapons/estaff_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/estaff_righthand.dmi'
+	name = "riot suppression electrostaff"
+	desc = "A large quarterstaff, with massive silver electrodes mounted at the end."
+	force = 10
+	damtype = BRUTE
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = ITEM_SLOT_BACK
+	sharpness = FALSE
+	force_unwielded = 5
+	force_wielded = 10
+	throwforce = 1
+	throw_speed = 1
+	block_chance = 50
+	materials = list(MAT_METAL = 4000, MAT_SILVER = 2000)
+	hitsound = 'sound/weapons/staff.ogg'
+	attack_verb = list("suppresed", "struck", "beaten", "thwacked", "pulped", "shocked")
+	total_mass = 5		//yeah this is a heavy thing, beating people with it while it's off is not going to do you any favors.
```

Comment:
> yeah this is a heavy thing, beating people with it while it's off is not going to do you any favors.

Not sure I understand what you're getting at with this comment; beating someone with a large, heavy stick will surely do you many favors, if you're trying to kill them or knock them into crit.

Reply:
this is the stamina drain on hit variable

if you try to start meleeing people to death without using harm intent you're going to run out faster than with stunbaton. this is to curb solo tase & lase playstyles.

Reply:
Ah okay, that makes sense